Add lore information on the New England Church Community into the design document (#66530)
CleverRaven/Cataclysm-DDA: doc/Lore/lore-factions.md

#### Goals
They believe that the Cataclysm is only a temporary state.

#### Size
22 people made up of a few religious leaders, their followers, and a few outsiders that stay around.

#### Attitudes
The NECC didn't expect outsiders to survive the Cataclysm, nevertheless they're generally friendly, but distant towards other survivors.  They are willing to offer shelter to anyone that needs it and are eager to trade medicinal herbs, raw materials, or small crafts for the right price. That's where their Good Samaritanism ends, however.

The NECC discriminates against anyone that's not part of their sect.  They are unwilling to let anyone in their "ark" – share their stockpile with outsiders or letting them join their congregation.  They put the blame on the victims for not joining their sect when they had the time – after all, the NECC did put out flyers urging people to join them while they still could.  Now the ark's gate has shut, and the rest must fend off for themselves.

They are strongly against anyone with bionics, or anyone who willingly mutated.

> Allied with Free Merchants, Tacoma Commune
> Friendly with Isherwood Family

Reviewer: ```suggestion
Knows of the Free Merchants and the Tacoma Commune and is friendly enough to trade, but like with all outsiders, keeps them at arms length.
Standoffish with the Isherwood Family.
```
They're a group of isolationists, and are always wary and suspicious of outsiders. They wouldn't be really friendly with any of these people beyond business interactions.

Author: @MNG-cataclysm  Thank you so much for the super fast suggestions!

Dislike the Old Guard (for being percieved as getting us into this mess) and any group that promotes bionics, mutations, or interactions with the nether creatures.
Hostile towards Hell's Raiders and Marloss Evangelists 

#### Bases
New England Church Retreat camp located in a forest.

#### Trade/Economics

> Miscellaneous, tools, supplies, and pelts.

Reviewer: ```suggestion
Miscellaneous, tools, supplies, and pelts. They use their own form of currency based on religious icons to manage internal trade, which are virtually worthless to outsiders. As the Cataclysm wears on, they may gain enough economic power to be worth something.
```
Should probably mention what currency they use, and which direction it could develop in.

Author: Added

#### Missions

> Helping with various tasks around the camp, and later personal matters of the members.

Reviewer: ```suggestion
Helping with various tasks around the camp, and later personal matters of the members. A trusted individual may be able to assist in outside politicking or spreading the word of God.
```
We should probably mention the direction they could develop in, if anyone read over this and decided to work on it.

Author: Added. My commit message is pretty bad, I hope I will be able to squash the commits, and add you at the end as an author
